mysql-test/t/percona_innodb_fake_changes.test

```diff
@@ -368,5 +372,53 @@ COMMIT;
 
 DROP TABLE t1, t3;
 
+#-------------------------------------------------------------------------------
+#
+# Scenario will try to insert data that will cause split of a node something
+# which will not happen in real if fake_changes = 1.
+#
+
+# clustered index split
+use test;
+create table t1 (i int, j blob, primary key pk(i)) engine=innodb;
+insert into t1 values (1, repeat('a', 8000));
+insert into t1 values (2, repeat('b', 8000));
+set session innodb_fake_changes = 1;
+--error ER_ERROR_DURING_COMMIT
+insert into t1 values (3, repeat('c', 8000));
```

Comment:
This seems to be dependent on row_format and page_size. Since we are looking to extend MTR testing to non-16k innodb_page_size pages, perhaps let's either rewrite this bit to work with all page sizes, or split to a separate 16k-requiring testcase?